sockets/delegateMonitor.js

```diff
@@ -81,7 +82,7 @@ module.exports = function (app, connectionHandler, socket) {
     // Private
 
     var log = function (msg) {
```

Comment:
That give use not so much flexibility, because we cannot specify log level and all logs in current file are logged with `info` level - to change, also in other files. `log` function should be removed or it should accept `level` as a param, and proper levels should be used.